src/app/actions/app.js

```diff
@@ -75,7 +75,6 @@ export const fetchAppRestrictions = () => (dispatch) => Promise.resolve()
       .catch((e) => {
         log.error(e)
         const message = 'Failed to fetch IP address restrictions'
-        dispatch(restrictionsError(message))
         throw new Error(message)
```

Comment:
I think this will still stop the app from loading. In chrome devtools try blocking the geoinfo url to simulate the scenario. If the geo request fails the app should still load